profiler/docdemo/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:alpine
```

Comment:
Is the Dockerfile used? We usually assume people have `go` installed.

Reply:
I included the Dockerfile only for consistency with other tools. However, am happy to remove it.